### PR TITLE
MPS: Fix some memory leak issues in release pools

### PR DESCRIPTION
**Pull request by kulinseth**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Review by albanD — APPROVED** [review]

File: aten/src/ATen/native/mps/OperationUtils.mm [hunk 1]
```diff
@@ -347,7 +347,7 @@ void printTensorNDArray(const Tensor& t) {
 
   _value = [[MPSGraphTensorData alloc] initWithMTLBuffer:selfBuf
```
Comment (janeyx99) [inline aten/src/ATen/native/mps/OperationUtils.mm:348]: Hey! missing [ here

**Comment by albanD** [comment]
Sorry about that...

File: aten/src/ATen/native/mps/OperationUtils.mm (hunk 1 above)
Comment (kulinseth) [reply aten/src/ATen/native/mps/OperationUtils.mm:348]: Yes.

**Pull request closed without merging**